packages/ember-data/lib/system/store.js

```diff
@@ -440,9 +440,32 @@ Store = Ember.Object.extend({
     store.find('person', { page: 1 });
     ```
 
-    This will ask the adapter's `findQuery` method to find the records for
-    the query, and return a promise that will be resolved once the server
-    responds.
+    This will call the private method `findQuery`, which delegates the query to the adapter. The adapter then makes a call to the server, which will return a `RecordArray` when the promise resolves.
```

Comment:
You're right, it wouldn't be good to document internal behavior. My intent was to clarify that by passing an object `{page: 1}` has an argument, it delegates to another method which ultimately makes a call to the server and transforms the object as parameters. How about this rephrasing: 

> By passing an object `{page: 1}` as an argument to the `find` method, it delegates to the adapter's `findQuery` method. The adapter then makes a call to the server, transforming the object `{page: 1}` as parameters that are sent along, and will return a `RecordArray` when the promise resolves.